packages/buidler-etherscan/src/ABIEncoder.ts

```diff
@@ -31,6 +33,11 @@ export async function encodeArguments(
   Encoder error reason: ${error.reason}`;
       throw new NomicLabsBuidlerPluginError(pluginName, message, error);
     }
+    if (isABIArgumentOverflowError(error)) {
+      const message = `Value ${error.value} is not a safe integer and cannot be encoded.
```

Comment:
We should add something like "Use a string instead of a plain number."

Reply:
I like that suggestion.

Reply:
Ah, of course. I forgot about that.